<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LOGIN</class>
 <widget class="QMainWindow" name="LOGIN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOGIN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>461</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QPushButton" name="Login_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>281</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>281</width>
       <height>90</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_ForgotMyPassword">
        <property name="text">
         <string>Forgot my Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_CreatAccount">
        <property name="text">
         <string>Create a New Account!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>281</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Login_Label">
        <property name="text">
         <string>username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Login_Input">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>281</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Password_Label">
        <property name="text">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Password_Input">
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
